docs/changelog.md

```diff
@@ -3,6 +3,11 @@ id: changelog
 title: Changelog
 sidebar_label: Changelog
 ---
+## v2.0.0 - Sep 17, 2021
+
+### Major Features and Improvements
+* Added support for parameters with `CREATE` clause.
```

Comment:
it's hard to figure out what you mean by this. Give an example.
"Added support for parameters with `CREATE` clause in the following form: `CREATE (n $param)`"